lib/Ast.ml

```diff
@@ -66,8 +36,14 @@ and color =
   | BrightCyan
   | BrightWhite
 
+(** [default_text_color] : Default Text Color *)
+let default_text_color = Black
```

Comment:
Pas plutôt l'inverse avec les terminales ? Blanc sur noir au lieu de noir sur blanc ?

Reply:
As you wish !

Reply:
Fixed in 7e2e69c3a78926680177f34b59ff62e5e6b840ee.